src/crates.rs

```diff
@@ -77,6 +78,11 @@ pub fn pre_fetch(config: &Config) {
     let config = config.clone();
     if let Some(_) = config.prefetch_path {
         let prefetch_path = config.prefetch_path.clone().unwrap();
+        let prefetch_ext = Path::new(&prefetch_path).extension().unwrap();
```

Comment:
I would rather this not unwrap; rather, the next line could read: `if prefetch_ext == Some("lock")` or some such, as a way to contain the rick of a user passing a file name in that doesn't have an extension.

Reply:
Good point, done.